cpp/include/cudf_test/column_wrapper.hpp

```diff
@@ -1595,7 +1596,12 @@ class lists_column_wrapper : public detail::column_wrapper {
     thrust::copy_if(
       std::cbegin(cols), std::cend(cols), valids, std::back_inserter(children), thrust::identity{});
 
-    auto data = children.empty() ? cudf::empty_like(expected_hierarchy) : concatenate(children);
+    // TODO: Once the public concatenate API exposes streams, use that instead.
```

Comment:
Can we track this TODO in an issue for stream APIs? I fear this will be lost.

Reply:
https://github.com/rapidsai/cudf/issues/13687